src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -24,52 +33,90 @@ export class VanguardArtistWrapper extends React.Component<
+        return GettingTheirDue(index)
+    }
+  }
+
   render() {
-    const { article } = this.props
+    const { article, section } = this.props
     const { hero_section, layout, slug, title } = article
     const { isExpanded } = this.state
+    const svgIndex = this.getRandomSVGIndex(section)
+    const svgString = encodeURIComponent(
+      ReactDOM.renderToStaticMarkup(this.getSVGBackground(
+        svgIndex,
+        section
+      ) as any)
+    )
+    const url = `url("data:image/svg+xml,${svgString}")`
 
     return (
-      <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
-        <Box textAlign="center">
-          <ArtistTitle size="8">{title}</ArtistTitle>
-          <Box position="absolute">
-            <Share
-              // TODO: We may need to use custom urls for in-page routing
-              url={getFullEditorialHref(layout, slug)}
-              title={title}
-            />
+      <SVGBackground url={url}>
+        <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
+          <Box textAlign="center" pt={50}>
+            <ArtistTitle size="8">{title}</ArtistTitle>
+            <Box position="absolute">
+              <Share
+                // TODO: We may need to use custom urls for in-page routing
+                url={getFullEditorialHref(layout, slug)}
+                title={title}
+              />
+            </Box>
+            {hero_section && (
+              <Sans size="4" weight="medium">
+                {hero_section.deck}
+              </Sans>
+            )}
           </Box>
-          {hero_section && (
-            <Sans size="4" weight="medium">
-              {hero_section.deck}
-            </Sans>
-          )}
-        </Box>
-        {/** TODO: Sections may need to be customized to handle expansion */}
-        <Sections article={article} customWidth={900} />
 
-        <ReadMoreButton
-          size="5"
-          weight="medium"
-          textAlign="center"
-          onClick={this.onExpand}
-        >
-          {isExpanded ? "Read More" : "Close"}
-          <Sans size="8">{isExpanded ? "\u2193" : "\u2191"}</Sans>
-        </ReadMoreButton>
-      </ArtistContainer>
+          {/** TODO: Sections may need to be customized to handle expansion */}
+          <Sections article={article} />
+
+          <ReadMoreButton
+            size="5"
+            weight="medium"
+            textAlign="center"
+            onClick={this.onExpand}
+          >
+            {isExpanded ? "Read More" : "Close"}
+            <Sans size="8">{isExpanded ? "\u2193" : "\u2191"}</Sans>
+          </ReadMoreButton>
+        </ArtistContainer>
+      </SVGBackground>
     )
   }
 }
 
 const ArtistTitle = styled(Serif)`
   font-size: 100px;
   line-height: 1.35em;
+  color: ${color("white100")};
 `
 
+const ReadMoreButton = styled(Sans)<{ onClick: () => void }>`
+  text-transform: uppercase;
+`
 const ArtistContainer = styled(Box)`
-  /* override feature text drop-caps */
```

Comment:
Why remove this comment?

Reply:
I just noticed that too, I think its a leftover when I was experimenting with bringing the container inside the component function.